teslatoken.py

```diff
@@ -21,9 +21,12 @@ def main():
     parser.add_argument("-o", "--output", metavar='FILE', default=None, help="Write to file FILE (json format)")
     parser.add_argument("-u", "--username", metavar='USER', default=None, help="Username for your MyTesla account")
     parser.add_argument("-p", "--password", metavar='PASS', default=None, help="Password for your MyTesla account")
+    parser.add_argument("-c", "--credential_file", metavar='CREDENTIAL_FILE', default=None, help="File which contains your MyTesla account password")
     parser.add_argument("-r", "--refresh", metavar='REFRESH_TOKEN', default=None, help="Refresh token for an access token")
     parser.add_argument("-q", "--quiet", action='store_true', help="Do not print token to stdout")
     parser.add_argument('--version', action='version', version='%(prog)s 0.1')
+    parser.add_argument('--revoke', action='store_true', help="Revokes a previously acquired access token")
+    parser.add_argument('-a', '--access_token', metavar='ACCESS_TOKEN', help="The access token to be revoked")
```

Comment:
I don't think the --access_token option is needed. Access token to revoke can be added to --revoke the same way as with --refresh. There will never be a case when using --revoke without a token.